src/porepy/numerics/fv/mpfa.py

```diff
@@ -1329,13 +1410,13 @@ def _create_bound_rhs(
         if subface_rhs:
```

Comment:
Could you by any chance deal with the "NOW WHAT?"?

Reply:
I wish. But since I have no idea what it meant when I wrote it some years ago, I deleted it.